permissions/jolla-camera.profile

```diff
@@ -1,16 +0,0 @@
-# -*- mode: sh -*-
-
-# Firejail profile for /usr/bin/jolla-camera
-
-# x-sailjail-translation-catalog =
-# x-sailjail-translation-key-description =
-# x-sailjail-description = Execute jolla-camera application
-# x-sailjail-translation-key-long-description =
-# x-sailjail-long-description =
-
-### PERMISSIONS
-# x-sailjail-permission = Camera,Pictures,Videos,MediaIndexing,Location,Privileged
-
-### APPLICATION
-private-bin jolla-camera,jolla-camera-lockscreen
```

Comment:
Does jolla-camera-lockscreen work now?

Reply:
Yes. These private-bin things are required if there is even one of them defined but when there is none, all binaries are available in the sandbox.